### PR TITLE
Updated CI signal handbook - filing issue

### PR DESCRIPTION
**Pull request by alejandrox1**
Updating the handbook with the procedure CI signal used this release for filling issues.

/area release-team
/cc @spiffxp @smourapina @kacole2 @mortent @jberkus 

Signed-off-by: Jorge Alarcon Ochoa <alarcj137@gmail.com>

### CODE REVIEW

File: release-team/role-handbooks/ci-signal/README.md [hunk 1]
```diff
@@ -201,44 +201,55 @@ CI signal is not currently using `priority/backlog` or `priority/awaiting-more-e
...
-...any additional debugging info I can offer...
-```
-Examples:
-- [[job failure] ci-kubernetes-e2e-kubeadm-gce](https://github.com/kubernetes/kubernetes/issues/54905)
+`[sig-network] Firewall rule [Slow] [Serial] should create valid firewall rules for LoadBalancer type service`
 
-If a test case is failing, file a v1.y milestone issue in kubernetes/kubernetes titled: `[e2e failure] full test case name`
-```
-/priority critical-urgent
+Include a spyglass link to the latest failed job and a triage link to visualize
```
Comment (mariantalla) [inline release-team/role-handbooks/ci-signal/README.md]: Consider linking to spyglass docs here.

Comment (mariantalla) [reply release-team/role-handbooks/ci-signal/README.md]: ...and to triage:)

**Comment by mariantalla** [comment]
Looks good! My only concern is that a good part of the advice on how to structure the issue is tied to the current version of the [issue template](https://github.com/kubernetes/kubernetes/blob/6a277e0c4dac8cce4a69cad91fcc7c65de32688c/.github/ISSUE_TEMPLATE/failing-test.md) for failing tests. This makes the handbook wordy and will cause it to go out of date if/when the template changes.

An alternative could be to say something like "include the following information: test name, prow job name, triage link, spyglass link, according to the failing test template". That's merely an alternative though; you'll be the one using the handbook so consider what will be more useful for you to have around.

**Comment by justaugustus** [comment]
@alejandrox1 -- any update on this handbook?

**Comment by alejandrox1** [comment]
gonna close this. Been working on a new version with the ci signal team - will create a new pr for that one

**Pull request closed without merging**